docs/package.json

```diff
@@ -17,9 +17,11 @@
     "gatsby-source-filesystem": "^2.0.28",
     "gatsby-transformer-remark": "^2.3.8",
     "lodash": "^4.17.11",
+    "lodash.debounce": "^4.0.8",
```

Comment:
looks like the full `lodash` library is already in the mix, so you probably don't need `lodash.debounce`

Reply:
I didn't konw that, uninstalled lodash.debounce!

Reply:
Oops sorry I missed that bit, I think we need the `lodash.debounce` separately. `lodash` is used only in built scripts, not in the frontend of the documentation. The `lodash.debounce` method is currently imported in `docs/src/components/search.js` so we have to keep it, even though it looks weird in package.json

Reply:
https://github.com/octokit/rest.js/blob/b8f2097ef7bf03ffde602a6ef866898ee025dbf3/docs/src/components/search.js#L2

I forgot to change import statement and pushed changed code. Is this okay with just `lodash` dependency?

Reply:
I’m not sure if gatsby does proper tree shaking to remove unused code, but it should be okay. Maybe compare the download size of JS assets locally to what we currently have in production? I thenk you have to run `gatsby build` and then `gatsby serve`. I don’t think `gatsby develop` will do all the bundling and minification

Reply:
Above import statement imports whole `lodash` library so changed it to import only a debounce function.
https://github.com/octokit/rest.js/blob/2bb9a17bd380bf0eaf9f1bf9baeb7d40d9bf357d/docs/src/components/search.js#L2

without debounce imported (24.21KB transferred and file size is 74.02KB):
![2019-06-04T15:57:49](https://user-images.githubusercontent.com/5436405/58858237-18e67c80-86e2-11e9-9a8b-15fddd683f99.png)

with debouce imported:
![2019-06-04T15:57:40](https://user-images.githubusercontent.com/5436405/58858236-184de600-86e2-11e9-839a-306e60c77738.png)

Is this affordable?

Reply:
There is probably use for optimization but that can be addressed in a future PR :)